frontend/components/ShareButton/index.jsx

```diff
@@ -0,0 +1,69 @@
+import React, { Component } from 'react';
+import ShareIcon from '@shopgate/pwa-ui-ios/icons/ShareIcon';
+import MaterialShareIcon from '@shopgate/pwa-ui-material/icons/MaterialShareIcon';
+import Ripple from '@shopgate/pwa-ui-shared/Ripple';
+import PropTypes from 'prop-types';
+import styles from './style';
+import getConfig from '../../helpers/getConfig';
+import connect from '../../connector';
+
+const { androidShareSVG } = getConfig();
+const { iOSShareSVG } = getConfig();
+/**
+ * ShareButton component
+ */
+class ShareButton extends Component {
+  static propTypes = {
+    iOSTheme: PropTypes.func.isRequired,
+    shareItem: PropTypes.func.isRequired,
+    shareParams: PropTypes.shape(),
+  };
+
+  static defaultProps = {
+    shareParams: null,
+  };
+  /**
+   * Handles the share button click
+   * Show's share screen for app
+   * @param {Object} event The click event object
+   */
+  handleClick = () => {
+    this.props.shareItem();
+  }
+  /**
+   * Renders the share icon depending on theme
+   * @returns {JSX}
+   */
+  renderIcon() {
+    if (this.props.iOSTheme()) {
+      return <ShareIcon svg={iOSShareSVG} />;
+    }
+    return <MaterialShareIcon svg={androidShareSVG} />;
+  }
+  /**
+   * Renders the components
+   * @returns {JSX}
+   */
+  render() {
+    if (!this.props.shareParams) {
```

Comment:
Button is shown even if deepLink is null (which makes it unusable, I believe).

Reply:
resolved issue with deepLink set to undefined